board/safety_declarations.h

```diff
@@ -69,6 +69,11 @@ typedef struct {
   const struct lookup_t angle_rate_up_lookup;
   const struct lookup_t angle_rate_down_lookup;
   const int max_angle_error;  // used to limit error between meas and cmd
+
+  const bool disable_rate_limits;
+  const bool enforce_angle_error;
+  const bool disable_near_angle_check;
```

Comment:
made the defaults the opposite of what ford uses, so no change in other safety modes are needed